http/action/src/main/java/io/knotx/databridge/http/action/ResponseOptions.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.databridge.http.action;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@DataObject(generateConverter = true, publicConverter = false)
+public class ResponseOptions {
+
+  private Set<String> predicates;
+  private boolean forceJson;
```

Comment:
I'm not sure if we need this option.

Reply:
if we remove this boolean, configs won't be read properly